### ISSUE TITLE
Keyboard and mouse are laggy when external displays are connected

### ISSUE THREAD

**Issue by rosencreuz** [poliva/lightum#47]
I've installed lightum a few days ago from the latest develop commit. It worked fine until I connected external monitors. Mouse freezes shortly every half a second or so. Same with keyboard when typing. When I kill the lightum process it's back to normal again. When I disconnect external monitors it's back to normal again.

I'm using Ubuntu 14.04 LTS with default configuration on MBP11.1.

**Comment by esoleyman** [comment]
From my understanding of the code, the application is not meant to work with multi-monitor displays. Is this how you have your system set up?

**Comment by rosencreuz** [comment]
I use it as my workstation during working hours during which I connect two external monitors. So half of the time I have 2 external monitors (lid closed), and the other half of the time lid open with no external monitors. I don't think this is a very exotic setup.

**Comment by esoleyman** [comment]
I'm not saying that it is an exotic setup but in reading the software it wasn't made to use multiple displays and as a result might not be dealing well with your specific setup.

For example, get_light_sensor_value() in functions.c reads from /sys/devices/platform/applesmc.768/light and get_screen_backlight_value() reads from /sys/class/backlight/acpi_video0.

These function are tied to the built-in display only. There are going to be problems with attempting to dim all of the other monitors. Please correct if I am wrong.

**Comment by rosencreuz** [comment]
That's right. I'm also not expecting it to control external monitors. I expect it only control the built-in display. But when built-in display is turned off it shouldn't cause problems. Also it should ignore external displays without causing problems.

**Comment by esoleyman** [comment]
We'll need to debug the code to figure out where the problem lies. Given that I don't have external monitors to attach to my laptop, I'll need time to figure it out. Any help will be appreciated.
